Svc/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/StaticMemory/")
 add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/TlmChan/")
 add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/TlmPacketizer/")
 add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/SystemResources/")
+add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/Ports/VersionPorts")
```

Comment:
Could be good to add a CMakeLists.txt under `Svc/Ports/` so that we can only include 
```
add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/Ports")
```
here, and future ports can easily be added under the Svc/Ports module

Reply:
Is this pointed to VersionPorts or are you saying in general?